openstack/nova/templates/_helpers.tpl

```diff
@@ -0,0 +1,51 @@
+{{- define "rabbitmq_user" -}}
+  {{.Values.rabbitmq.users.default.user}}
+{{- end -}}
+
+{{- define "rabbitmq_password" -}}
+  {{ .Values.rabbitmq.users.default.password | default (tuple . .Values.rabbitmq.users.default.user | include "rabbitmq.password_for_user")}}
+{{- end -}}
+
+{{- define "cell0_db_path" -}}
+{{.Values.cell0dbUser}}:{{.Values.cell0dbPassword | default .Values.apidbPassword | urlquery}}@{{.Chart.Name}}-postgresql.{{include "svc_fqdn" .}}:5432/{{.Values.cell0dbName}}
+{{- end -}}
+
+{{- define "rabbit_url" -}}
+  {{- $rabbitmq_host := printf "nova-rabbitmq.%s" ( include "svc_fqdn" . ) -}}
+  rabbit://:{{ include "rabbitmq_password" . | urlquery}}@{{ $rabbitmq_host }}:5672/
+{{- end -}}
+
+{{- define "container_image_nova" -}}
+  {{- $name := index . 1 -}}
+  {{- with index . 0 -}}
+    {{- $version_name := printf "imageVersionNova%s" ($name | lower | replace "-" " " | title | nospace) -}}
+    {{- $image_name := ( .Values.loci.nova | ternary .Values.imageNameNova (printf "ubuntu-source-nova-%s" ($name | lower)) ) -}}
```

Comment:
Here's a loci reference. Didn't loci-support get reverted? Was it reverted only in parts?

Reply:
Oh. I think this is the one that was included for "loci support" and is thus currently unused.

Reply:
exactly, since I leave it since I want to reimplement it as soon the pipelines are adapted